newIDE/app/src/Hints/ObjectsAdditionalWork.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+import { t } from '@lingui/macro';
+import { type AlertMessageIdentifier } from '../MainFrame/Preferences/PreferencesContext';
+import newNameGenerator from '../Utils/NewNameGenerator';
+
+/*
+ * Define additional logic which executes after an object/instance has been created.
+ * Also, InfoBar can be used which could notify users of the additional changes.
+ * Declare new identifier for infoBar in Mainframe/Preferences/PreferenceContext
+ * and add it in hints/explaination list.
+ */
+
+export type InfoBarDetails = {
+  show: boolean,
```

Comment:
Should show even be in this object type? It's a responsibility of the editor I think.